server/src/main/auth/auth.routing.ts

```diff
@@ -0,0 +1,26 @@
+import { Router } from "express";
+import { register } from "./register/register.controller";
+import { login } from "./login/login.controller";
+import { verify } from "./verify/verify.controller";
```

Comment:
register、login、verify 這三個可以不用個別是一個 controller 層級，而是一個 controller 底下的不同個 action 就好。
因為他們三個邏輯應該蠻息息相關的。

```suggestion
import controller from "./auth.controller";
// controller.register
// controller.verify
// controller.login
```